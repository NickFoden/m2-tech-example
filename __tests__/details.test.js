import renderer from "react-test-renderer";
import Detail from "../components/detail"

describe("Details component", () => {
    test("smoke", () => {
        const snap = renderer.create(
            <Detail />
            ).toJSON();
            expect(snap).toMatchSnapshot();
        })
})