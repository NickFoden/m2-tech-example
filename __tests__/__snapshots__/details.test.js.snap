// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details component smoke 1`] = `
<div>
  No account provided
</div>
`;
